test/libsolidity/ABIEncoderTests.cpp

```diff
@@ -369,7 +357,7 @@ BOOST_AUTO_TEST_CASE(external_function_cleanup)
 	)";
 	BOTH_ENCODERS(
 		compileAndRun(sourceCode);
-		callContractFunction("f(uint256)");
+		callContractFunction("f(uint256)", u256(0));
```

Comment:
Just remind me: does the newer decoder changes semantics and requires the minimum length of the encoding present as calldata?

Do we have a specific test for short data with the new decoder?

Reply:
Yes, the new decoder does not tolerate short data and there are multiple tests.